### ISSUE TITLE
IRNet cannot run after update

### ISSUE THREAD

**Issue by CindyTing** [microsoft/IRNet#24]
Hi, I downloaded IRNet's last version  and ran it . However, it has a running error like this:
![image](https://user-images.githubusercontent.com/35062414/75514314-2a22a680-5a32-11ea-8236-96ca71475cc1.png)
it seems caused by pytorch version, and i updated  it to torch 1.4, and then it has another error:
![image](https://user-images.githubusercontent.com/35062414/75514403-5d653580-5a32-11ea-8c4c-8d9cc53e72bd.png)
how can we solve this error ?

thank you for your help

**Comment by shubhamk16** [comment]
try to install it from following conda command:

conda install pytorch torchvision cudatoolkit=9.2 -c pytorch -c defaults -c numba/label/dev

**Comment by jaydeepb-inexture** [comment]
> try to install it from following conda command:
> 
> conda install pytorch torchvision cudatoolkit=9.2 -c pytorch -c defaults -c numba/label/dev

i was also getting the same error but after doing setting pytorch by above command model is training but i got some weird userwarnings?

i have read your issue https://github.com/microsoft/IRNet/issues/23.
can i got direction how to implement the BERT in current IRNet version.

**Comment by jaydeepb-inexture** [comment]
> try to install it from following conda command:
> 
> conda install pytorch torchvision cudatoolkit=9.2 -c pytorch -c defaults -c numba/label/dev

i was also facing the above error but using same command mention above it was solved.

**Comment by shantanu-kumar242** [comment]
Thanks a lot. I went through output.log file and got this error even after running "conda install pytorch torchvision cudatoolkit=9.2 -c pytorch -c defaults -c numba/label/dev" command. I am running this model on cpu as my system doesn't have gpu. :
![issue1](https://user-images.githubusercontent.com/51315728/75963222-19b77380-5eeb-11ea-91b9-7f5d4b7a4b3a.png)

**Comment by jaydeepb-inexture** [comment]
@shantanu-kumar242  are using linux or windows?

**Comment by 489597448** [comment]
remove  all boo()

**Comment by anshudaur** [comment]
HI, 
I am trying to run the code on colab. But i am not able to because of pytorch 1.0.0 deprecation warning :
Prior to PyTorch 1.1.0, the learning rate scheduler was expected to be called before the optimizer’s update; 1.1.0 changed this behavior in a BC-breaking way. If you use the learning rate scheduler (calling scheduler.step()) before the optimizer’s update (calling optimizer.step()), this will skip the first value of the learning rate schedule. If you are unable to reproduce results after upgrading to PyTorch 1.1.0, please check if you are calling scheduler.step() at the wrong time.

I was able to run the IRNET+bert on colab but not this one. I tried dowgrading the torch version to : 0.4.1 but that also didn't make it work. 
Thanks
Anshu
